modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/extensions/Fragment.kt

```diff
@@ -0,0 +1,16 @@
+package au.com.shiftyjelly.pocketcasts.compose.extensions
+
+import androidx.compose.foundation.layout.consumeWindowInsets
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.ComposeView
+import androidx.fragment.app.Fragment
+import androidx.fragment.compose.content
+
+// don't consume the insets or older versions of Android such as SDK 26 it will effect the layout of other areas of the app such as the What's New bottom sheet
+fun Fragment.contentWithoutConsumedInsets(contentWithoutInsets: @Composable () -> Unit): ComposeView {
```

Comment:
Do you think I should inline this function?

Reply:
No there's no need. This isn't a part of a library so we don't really need to be concerned about the method count.